packages/server/app/Controllers/Http/SearchUsersController.ts

```diff
@@ -87,14 +87,19 @@ export default class SearchUsersController {
             throw new ForbiddenException();
         }
 
-        await User.findOrFail(tmpAuthUserID);
-        //TODO refactor after follow feature implem
-        const requestingUserIsfollowingRelatedUser = false; //tmp
+        const requestingUser = await User.findOrFail(tmpAuthUserID);
+        await requestingUser.load('following', (userQuery) => {
+            return userQuery.where('uuid', userID);
+        });
+        const requestingUserIsfollowingRelatedUser =
+            requestingUser.following.length > 0;
 
         const relateduser = await User.findOrFail(userID);
         await relateduser.load('playlistsVisibilitySetting');
         await relateduser.load('relationsVisibilitySetting');
         await relateduser.load('mpeRooms');
+        await relateduser.load('following');
+        await relateduser.load('followers');
```

Comment:
If we only want the count of following or followers, maybe could we use [`relatedUser.loadCount()`](https://docs.adonisjs.com/guides/models/relationships#lazy-load-relationship-aggregates) to do not load the information of the users?